HW03/aleksei_zhuchkov/aleksei_zhuchkov_hw_03_t_04.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class Homework3
+  def task4(data_log)
+    raise ArgumentError unless data_log.instance_of?(String)
+
+    count_digits = data_log.count '0-9'
+    count_letters = data_log.downcase.count 'a-z'
```

Comment:
```suggestion
    count_letters = data_log.count('a-zA-Z')
```

Reply:
@B0RlS  I have done it. Please check it.

Reply:
My last commit with changes https://github.com/labs-ruby/homeworks-2022/pull/63/commits/dafca72603fd958d058fbc5630fb9b7684cd8556